crates/rpc/eth/block.rs

```diff
@@ -276,6 +276,25 @@ pub fn block_number(storage: Store) -> Result<Value, RpcErr> {
     }
 }
 
+pub fn get_blob_base_fee(storage: &Store) -> Result<Value, RpcErr> {
+    info!("Requested blob gas price");
+    match storage.get_latest_block_number() {
+        Ok(Some(block_number)) => {
+            let header = match storage.get_block_header(block_number)? {
```

Comment:
shouldn't you be able to just do let `header = match storage.get_block_header(block_number)?`?

Reply:
You mean that we should return just `Ok(block_header)` instead of `Ok(Some(block_header))` from Storage?

Reply:
See https://github.com/lambdaclass/ethereum_rust/blob/main/crates/rpc/eth/block.rs#L93

Reply:
The `match` in that case is two lines below that one 🤔